docs/produce-events.md

```diff
@@ -236,10 +234,6 @@ To register your topic with with the Hub:
     **forwarders [optional]**
     Displayed in details page. Object with key-value pairs of `systemname: teamname`. Used if there is a system sitting in between data store and event bus which that mutates data before an event is published.
 
-    **payload [required]**
-    The structure of an event that gets published to a Kafka topic in Avro format. Should include all key names and their data type (string, int, etc).
-    <mark>Producers are responsible for providing accurate representations of their event payload</mark>.
-
     The catalog.yaml file will be validated against [this JSON schema](https://github.com/department-of-veterans-affairs/ves-event-bus-backstage-plugins/blob/main/plugins/event-kind-backend/src/schema/Event.schema.json).
```

Comment:
Since `schema` is included in this schema, it might be useful to add a note that this is auto-populated. Otherwise there could be some confusion about what `schema` is and if there's just a mismatch between our documentation and the source of truth.